packages/x-components/src/plugins/x-plugin.ts

```diff
@@ -233,8 +233,8 @@ export class XPlugin implements PluginObject<XPluginOptions> {
     if (!this.installedXModules.has(xModule.name)) {
       const customizedXModule = this.customizeXModule(xModule);
       this.registerStoreModule(customizedXModule);
-      this.registerWiring(customizedXModule);
       this.registerStoreEmitters(customizedXModule);
+      this.registerWiring(customizedXModule);
```

Comment:
We might want to comment the reasoning behind this change. Either an inline comment here or a `@remarks` section in the `registerXModule(...)` method documentation would suffice.